react-js/react.md

```diff
@@ -4,21 +4,22 @@
 8 Hours
 
 ### Prerequisites
-* [JavaScript I - VI](../javascript)
-* [jQuery Topic Outline](/web/jquery.md)
+* [HTML](../web/html.md)
+* [JavaScript 6 - OOP or above](../javascript/javascript-7-oop.md)
 
 ### Motivation
-I want to create interactive UIs more easily and with less code than jQuery.
+React.js is the most popular front-end framework for Web applications. 
 
-React is complex, but one reason it is awesome is that you can build a component with it's own html, css, and js characteristics - for example, a styled list item - and that's the only time you have to write it.  You can use it whenever you want by calling the component name you made up - <ListItem/> - anywhere you like in your project. Another reason is because you see changes instantly as you develop and as your app runs.  No more refreshing!
+React.js is an open-source JavaScript library that is used for building user interfaces specifically for single-page applications. It’s used for handling the view layer for web and mobile apps. React also allows us to create reusable UI components.
```

Comment:
Please find & replace "React.js". We don't need to specify that 😄

Reply:
It's not only for single-page applications so please remove that statement.

Reply:
Okay